src/common/common.h

```diff
@@ -97,13 +97,17 @@
  *  and is responsible for freeing it. */
 #define OWNER
 
+/** Structs that are passed over the wire should be annotated with this
+ *  attribute to make sure there is no padding. */
+#define SERIALIZABLE __attribute__((packed))
```

Comment:
Technically things without this attribute are serializable too, right? Would `PACKED_LAYOUT` or something be better?